pkg/odo/cli/url/list.go

```diff
@@ -92,6 +92,13 @@ func (o *ListOptions) Run(cmd *cobra.Command) (err error) {
 		outOfSync := false
 		for _, u := range urls.Items {
 			if u.Spec.Kind == localConfigProvider.ROUTE {
+				if u.Spec.Protocol == "" {
```

Comment:
Why are we setting the protocol in the CLI layer instead of the url package?

Reply:
Removed